lib/launch/launch_web.dart

```diff
@@ -13,9 +24,54 @@ class WebLaunchType extends LaunchType {
   bool canLaunch(String path, LaunchData data) => path.endsWith('.html');
 
   Future<Launch> performLaunch(LaunchManager manager, LaunchConfiguration configuration) {
-    // TODO:
-    return new Future.error(new UnimplementedError());
+    Browser browser = deps[BrowserManager].browser;
+    if (browser == null) {
+      atom.notifications.addWarning('No browser configured.');
+      return new Future.value();
+    }
+
+    // Find pub serve for 'me'.
+    ServeLaunch pubServe = manager.launches.firstWhere((l) =>
```

Comment:
You could also debug more than one html from the same project, having one pub serve for that use case is nice.
I'll add that as an option once the debugger works.  For now it's easier to build small steps.